internal/engine/allexperiments.go

```diff
@@ -85,10 +90,6 @@ var experimentsByName = map[string]func(*Session) *ExperimentBuilder{
 		}
 	},
 
-	// TODO(bassosimone): when we can set experiment options using the JSON
-	// we need to get rid of all these multiple experiments.
-	//
-	// See https://github.com/ooni/probe-engine/issues/413
```

Comment:
This was a lost battle. It's quite convenient to have these variations of the "example" experiment.